### DISCUSSION TITLE
User not set after loginWithProvider

### DISCUSSION THREAD

**Discussion by Tolli81** [Intevel/nuxt-directus#200] [Q&A] [answered]
Hi!

I have Directus working with Google authentication, and now I'm trying to piggyback on that in a nuxt app for the frontend.  To me it looks like the login is actually succeeding, but the `user` object isn't populated.

I'm running `nuxt 3.65` and `nuxt-directus 5.5.0`

What I've done so far to debug:
1. Make sure I can log in to Directus with Google account 
2. Logout from Directus
3. Open Chrome dev tools and record network traffic
4. In Nuxt app, trigger Login with Google (`@click="loginWithProvider('google')"`)
5. Verify correct request chain; directus google endpoint -> google auth endpoint -> directus callback -> nuxt app

Based my implementation off of `nuxt-directus-auth-example` and updated according to latest `nuxt-directus` docs.
It might be worth mentioning that I'm developing in GitHub Codespace.

![image](https://github.com/Intevel/nuxt-directus/assets/5773381/b1972fed-1b58-4da2-8a3e-efab32ddb2bb)

Any ideas?

-Tolli

**Comment by alexey-yarmosh** [comment]
Any updates on that?

**Reply by casualmatt** [reply, marked as answer]
Hi @alexey-yarmosh 5.X dosen't offically support SSO. 
Try to implement it with the nuxt-directus-next AKA 6.X, that support SSO

**Comment by casualmatt** [comment]
The issue is related to 5.x
